tests/bluetooth/tester/src/btp/btp_bas.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2024 Demant A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef BTP_BAS_H_
+#define BTP_BAS_H_
+
+#include <stdint.h>
+#include <stddef.h>
+
+/* BTP BAS Service Opcodes */
+#define BTP_BAS_SET_BATTERY_LEVEL             0x02
+#define BTP_BAS_BLS_SET_BATTERY_PRESENT       0x03
+#define BTP_BAS_BLS_SET_WIRED_POWER_SOURCE    0x04
+#define BTP_BAS_BLS_SET_WIRELESS_POWER_SOURCE 0x05
+#define BTP_BAS_BLS_SET_BATTERY_CHARGE_STATE  0x06
+#define BTP_BAS_BLS_SET_BATTERY_CHARGE_LEVEL  0x07
+#define BTP_BAS_BLS_SET_BATTERY_CHARGE_TYPE   0x08
+#define BTP_BAS_BLS_SET_CHARGING_FAULT_REASON 0x09
+#define BTP_BAS_BLS_SET_IDENTIFIER            0x0A
+#define BTP_BAS_BLS_SET_SERVICE_REQUIRED      0x0B
+#define BTP_BAS_BLS_SET_BATTERY_FAULT         0x0C
+
+/* Command structures */
+struct btp_bas_set_battery_level_cmd {
+	bt_addr_le_t address;
+	uint8_t level;
+} __packed;
+
+struct btp_bas_bls_set_battery_present_cmd {
+	bt_addr_le_t address;
+	uint8_t present;
+} __packed;
+
+struct btp_bas_bls_set_wired_power_source_cmd {
+	bt_addr_le_t address;
+	uint8_t source;
+} __packed;
+
+struct btp_bas_bls_set_wireless_power_source_cmd {
+	bt_addr_le_t address;
+	uint8_t source;
+} __packed;
+
+struct btp_bas_bls_set_battery_charge_state_cmd {
+	bt_addr_le_t address;
+	uint8_t state;
+} __packed;
+
+struct btp_bas_bls_set_battery_charge_level_cmd {
+	bt_addr_le_t address;
+	uint8_t level;
+} __packed;
+
+struct btp_bas_bls_set_battery_charge_type_cmd {
+	bt_addr_le_t address;
+	uint8_t type;
+} __packed;
+
+struct btp_bas_bls_set_charging_fault_reason_cmd {
+	bt_addr_le_t address;
+	uint8_t reason;
+} __packed;
+
+#if defined(CONFIG_BT_BAS_BLS_IDENTIFIER_PRESENT)
+struct btp_bas_bls_set_identifier_cmd {
+	bt_addr_le_t address;
+	uint16_t identifier;
+} __packed;
+#endif /* CONFIG_BT_BAS_BLS_IDENTIFIER_PRESENT */
+
+#if defined(CONFIG_BT_BAS_BLS_ADDITIONAL_STATUS_PRESENT)
+struct btp_bas_bls_set_service_required_cmd {
+	bt_addr_le_t address;
+	uint8_t service_required;
+} __packed;
+
+struct btp_bas_bls_set_battery_fault_cmd {
+	bt_addr_le_t address;
+	uint8_t battery_fault;
+} __packed;
+#endif /* CONFIG_BT_BAS_BLS_ADDITIONAL_STATUS_PRESENT */
```

Comment:
```suggestion
struct btp_bas_bls_set_identifier_cmd {
	bt_addr_le_t address;
	uint16_t identifier;
} __packed;

struct btp_bas_bls_set_service_required_cmd {
	bt_addr_le_t address;
	uint8_t service_required;
} __packed;

struct btp_bas_bls_set_battery_fault_cmd {
	bt_addr_le_t address;
	uint8_t battery_fault;
} __packed;
```

No value in guarding definitions :)